src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir", normal, sans-serif;
+  font-weight: 300;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: url(images/Search.svg) no-repeat ;
+  outline: none;
+}
+
+.big_search_bar {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.big_input {
+  background-position: top 25px left 26px;
+  background-size: 19px 19px;
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+}
+
+.small_input {
+  background-position: top 15px left 13px;
+  background-size: 11px 11px;
+  padding-left: 33px;
+  padding-top: 12px;
+  padding-bottom: 10px;
+}
+
+input::placeholder {
+  font-family: inherit;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.big_input::placeholder {
```

Comment:
add font-weight 300 on the placeholder, it should not be bold on focus, only input text